vsts/scripts/removeDockerArtifacts.sh

```diff
@@ -27,5 +27,5 @@ echo
 if [ -d "$mountedDirs" ]; then
     echo
     echo "Cleaning up files created by test containers ..."
-    docker run -v $mountedDirs:/tempDirs oryxdevmcr.azurecr.io/public/oryx/build /bin/bash -c "rm -rf /tempDirs/* && ls /tempDirs"
+    docker run -v $mountedDirs:/tempDirs alpine /bin/sh -c "rm -rf /tempDirs/* && ls /tempDirs"
```

Comment:
i found we are downloading big and bulky build image to clean up docker artifacts when we are doing clean up in runtime image job, so I changed this to alpine .. its just 5 mb ... so even though we have to pull it it should still be faster